solr/CHANGES.txt

```diff
@@ -56,6 +56,8 @@ Deprecation Removals
   end in "/solr").  Users may still specify a default collection through use of the `withDefaultCollection` method available
   on most SolrClient builders. (Jason Gerlowski, David Smiley, Eric Pugh)
 
+* SOLR-12089: Remove deprecated `breakSugestionTieBreaker` parameter in favor of `breakSuggestionTieBreaker`
```

Comment:
I added this under `Deprecation Removals` for 10.x - is there any other place to mention the change?

Reply:
There is another `major-changes-in-10.adoc`, but this doesn't quite seem like that.   And maybe we update that doc closer to 10 anyway!